Lib/test/_test_multiprocessing.py

```diff
@@ -659,20 +659,24 @@ def test_close(self):
         self.assertEqual(p.is_alive(), True)
         # Child is still alive, cannot close
         with self.assertRaises(ValueError):
+            self.assertFalse(p.closed)
             p.close()
+            self.assertTrue(p.closed)
```

Comment:
```suggestion
		self.assertFalse(p.closed)
        with self.assertRaises(ValueError):
            p.close()
```

`p.close()` raises ValueError so `self.assertTrue(p.closed)` will never be executed (and the assertion would be wrong in this case)